docs/api/cli.md

```diff
@@ -3,4 +3,373 @@ layout: docu
+
+The DuckDB CLI is based on the SQLite command line shell, so CLI-client-specific functionality is similar to what is described in the [SQLite documentation](https://www.sqlite.org/cli.html) (although DuckDB's SQL syntax follows PostgreSQL conventions).
+
+
+## Getting Started
+Once the CLI executable has been downloaded, unzip it and save it to any directory. Navigate to that directory in a terminal and enter the command `duckdb` to run the executable. If in a PowerShell environment, use the command `./duckdb` instead. To see additional command line options to use when starting the CLI, use the command `duckdb --help`.
+By default, the CLI will open a temporary in-memory database. To open or create a persistent database, simply include a path as a command line argument like `duckdb path/to/my_database.duckdb`. This path can point to an existing database or to a file that does not yet exist and DuckDB will open or create a database at that location as needed. The file may have any arbitrary extension, but `.db` or `.duckdb` are two common choices. You will see a prompt like the below, with a D on the final line.
+
+```command
+v0.3.4 662041e2b
+Enter ".help" for usage hints.
+Connected to a transient in-memory database.
+Use ".open FILENAME" to reopen on a persistent database.
+D
+```
+
+Once the CLI has been opened, enter a SQL statement followed by a semicolon, then hit enter and it will be executed. Results will be displayed in a table in the terminal. If a semicolon is omitted, hitting enter will allow for multi-line SQL statements to be entered. 
+
+```sql
+D SELECT 'quack' AS my_column;
+```
+
+| my_column |
+|-----------|
+| quack     |
+
+```sql
+D SELECT
+>     'nicely formatted quack' AS my_column,
+>     'excited quacking' AS another_column;
+```
+
+|       my_column        |  another_column  |
+|------------------------|------------------|
+| nicely formatted quack | excited quacking |
+
+The CLI supports all of DuckDB's rich SQL syntax including `SELECT`, `CREATE`, and `ALTER` statements, etc. 
+
+To exit the CLI, press Ctrl-C. If using a persistent database, it will automatically checkpoint (save the latest edits to disk) and close. This will remove the .WAL file (the Write-Ahead-Log) and consolidate all of your data into the single file database.
```

Comment:
Ctrl-D is friendlier.

Reply:
Hmm... Ctrl-D does not seem to work for me on Windows. Is it a Mac/Linux thing?

Reply:
Cntrl-D sends EOF, which when send on an empty line, is usually interpreted by the shell as a desire to exit.
Cntrl-C is akin to Alt-F4 and used to force quit, usually used to end an unresponsive process.